server/server.js

```diff
@@ -1,4 +1,11 @@
 const app = require('./index.js');
+// const compression = require('compression');
+// const expressStaticGzip = require('express-static-gzip');
+
+// app.use(compression());
+// app.use(expressStaticGzip(__dirname + '/../client/dist', {
+//   enableBrotli: true
+// }));
```

Comment:
If things are working, I think you could take out all commented out code. For the main repo, I don't think we'll want to have any commented out code.

Reply:
my one thought on it is that may be what we use when we stop using the webpack dev server.